### DISCUSSION TITLE
Teamviewer Client for Synology

### DISCUSSION THREAD

**Discussion by Gunnar-GO** [rustdesk/rustdesk#1233] [General]
Is there a way to deploy a client to the Synology NAS? 

This would be analog to the TeamViewer Client (https://www.teamviewer.com/en/integrations/synology/), it allows to collect the Devices at one place in the adressbook.

**Comment by schneider82** [comment]
This would be a huge deal for me! This is the reason we have to stay with TeamViewer. Can I give you guys money instead? How much would it cost to make this feature?

**Comment by youhers** [comment]
Development is not a simple matter

**Reply by schneider82** [reply]
Not sure what the point of that comment was... humans breath air.

**Comment by bigretromike** [comment]
Wouldn't it be simpler to just install VPN on that Synology and VPN clients on your users and they would use browser to access resourse ? from what I read TV integrated with synology user accounts as it works as RDP when multisession is available etc.

**Reply by schneider82** [reply]
It's far harder to manage and creates security issues. TeamViewer allows us to have some control and auditing of the connections made. Rustdesk would be a far better solution as it could be set up to only appear as our system, and I would like to support this software. It looks like a solution to TeamViewer's grip on the IT world (or at least our small part).
